readme.md

```diff
@@ -3,8 +3,53 @@
 This package contains a parser for the Checker Framework's stub files:
 https://checkerframework.org/manual/#stub
 It is a fork of the [JavaParser](http://javaparser.org) project.
-The remainder of this README file is the JavaParser README.
 
+## Differences between the StubParser and JavaParser
+
+Following list is the differences that were added to the JavaParser to adjust it for using as Stubparser for [the Checker Framework](https://github.com/typetools/checker-framework).
+
+1. StubUnit class that represents the parsed [stubfile](https://checkerframework.org/manual/#stub).
+2. Changes at the java.jj file to parse the stubfiles.
+3. Methods for parsing the stubfiles at the JavaParser.class.
+
+## Updating from upstream JavaParser
+
+This section describes how to incorporate changes from JavaParser into
+StubParser.  Only developers, not users, of StubParser need to do this.
+
+1. Fork [the stubparser project](https://github.com/typetools/stubparser) to your github account.
+2. Enable Travis build for your fork of the stubparser.

```

Comment:
It would be nice to say how to do this, even if just to link to https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI .